OSeMOSYS_PuLP_HP.py

```diff
@@ -398,17 +397,20 @@ def saveResultsToExcel(dataframe, fileDir, fileName):
 DaySplit = createParameter(df, 'DaySplit')
```

Comment:
You could iterate over a list of parameters and store the resulting initialised parameters in a dict rather than individual variables.